client/src/components/Visualizations/VisualizationCreate.vue

```diff
@@ -54,20 +57,37 @@ onMounted(() => {
 </script>
 
 <template>
-    <BAlert v-if="errorMessage" variant="danger" show>{{ errorMessage }}</BAlert>
-    <LoadingSpan v-else-if="!currentHistoryId || !plugin" message="Loading visualization" />
-    <div v-else>
-        <VisualizationHeader :plugin="plugin" />
-        <SelectionField
-            class="my-3"
-            object-name="Select a dataset..."
-            object-title="Select to Visualize"
-            object-type="history_dataset_id"
-            :object-query="doQuery"
-            @change="onSelect" />
-        <MarkdownDefault v-if="plugin.help" :content="plugin.help" />
-        <div v-for="(sample, sampleIndex) in samples" :key="sampleIndex">
-            {{ sample }}
+    <FormCardSticky
+        v-if="plugin"
+        :error-message="errorMessage"
+        :description="plugin?.description"
```

Comment:
```suggestion
        :description="plugin.description"
```

The `v-if="plugin"` guarantees that plugin will not be undefined, so you can safely remove all optional changing operators in this component's props.